cmd/antrea-agent/agent.go

```diff
@@ -250,9 +252,20 @@ func run(o *Options) error {
 		isChaining,
 		routeClient,
 		networkReadyCh)
-	err = cniServer.Initialize(ovsBridgeClient, ofClient, ifaceStore, entityUpdates)
-	if err != nil {
-		return fmt.Errorf("error initializing CNI server: %v", err)
+
+	cniPodInfoStore := cnipodcache.NewCniPodInfoStore()
```

Comment:
Probably assign cniPodInfoStore to cnipodcache.NewCniPodInfoStore inside the "if features.DefaultFeatureGate.Enabled(features.SecondaryInterfaceSupport)" block.